src/main/java/service/FruitServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package service;
+
+import dao.FruitQuantityStorageDao;
+import dao.FruitQuantityStorageDaoImpl;
+import java.util.List;
+import java.util.stream.Collectors;
+import model.FruitTransaction;
+import strategy.OperationStrategy;
+
+public class FruitServiceImpl implements FruitService {
+    private FruitQuantityStorageDao storageDao;
+    private OperationStrategy operationStrategy;
+
+    public FruitServiceImpl(OperationStrategy operationStrategy) {
+        this.operationStrategy = operationStrategy;
+        this.storageDao = new FruitQuantityStorageDaoImpl();
+    }
+
+    @Override
+    public List<FruitTransaction> getReport(List<FruitTransaction> transactions) {
```

Comment:
get String from transactions. What is the point of this if you process transactions in the writeToFileCsv method